### ISSUE TITLE
[a11y][RangeSlider]: range slider is not visible in high contrast mode

### ISSUE THREAD

**Issue by ginabak** [Shopify/polaris#3996]
## Issue summary

In high contrast mode, the range slider is not visible and hard to see (not accessible). 

## Expected behavior

The slider should be clear in high contrast mode

## Actual behavior

<details>
      <summary>Summary of your screenshot in high contrast mode (expand)</summary>

### Default Range Slider (no action)
![Default Range Slider (no action)](https://user-images.githubusercontent.com/43223543/107704337-15631100-6c8b-11eb-9e4f-8c7e51b398d4.jpg)

### Default Range Slider (in action - while sliding)
![default Range Slider (in action - while sliding)](https://user-images.githubusercontent.com/43223543/107704552-6c68e600-6c8b-11eb-8edb-a4ecf5b51722.jpg)

### Dual Thumb Range Slider (no action)
![ Dual Thumb Range Slider (no action)](https://user-images.githubusercontent.com/43223543/107704551-6c68e600-6c8b-11eb-8a7b-306bdabfab18.jpg)

### Dual Thumb Range Slider (in action - while selected)

![Dual Thumb Range Slider (in action - while selected)](https://user-images.githubusercontent.com/43223543/107704829-df725c80-6c8b-11eb-8a7a-737ea4ffc5f6.jpg)

 </details>

## Steps to reproduce the problem

1. Open Windows VM
2. Enable high contrast mode
3. Navigate to http://polaris-react.herokuapp.com/iframe.html?id=all-components-range-slider--all-examples&viewMode=story 

## Specifications

Paste the results here:

```bash

 System:
    OS: macOS 11.2
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 84.95 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 10.15.0 - ~/.nvm/versions/node/v10.15.0/bin/node
    Yarn: 1.13.0 - ~/.dev/yarn/v1.13.0/bin/yarn
    npm: 6.4.1 - ~/.nvm/versions/node/v10.15.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 88.0.4324.150
    Firefox: 85.0.2
    Safari: 14.0.3
  npmPackages:
    react: ^16.9.0 => 16.9.0
    react-dom: ^16.9.0 => 16.9.0

```
